src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.ecommercee.controller;
+
+public class UserController {
```

Comment:
Ten PR z UserController jeszcze nie został zmerge'owany a już mamy ten kontroller przeniesiony do innego pakietu. 
https://github.com/pplutap/JDP-1901-01/pull/4/files
Trzeba to jakoś ogarnąć. Dogadajcie się ze sobą :)